### PR TITLE
size: add page

### PR DESCRIPTION
**Pull request by suyashbhawsar**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/size.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# size
+
+> Displays the sizes of sections inside binary files.
+> More information: <https://www.ibm.com/docs/en/aix/7.2?topic=s-size-command>.
```
Comment (pixelcmtd) [inline pages/linux/size.md:4]: afaik aix is not linux
```suggestion
> More information: <https://sourceware.org/binutils/docs/binutils/size.html>.
```

File: pages/linux/size.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+- Display the size of sections of object or executable file:
+
+`size {{file}}`
```
Comment (pixelcmtd) [inline pages/linux/size.md:8]:
```suggestion
`size {{path/to/file}}`
```

File: pages/linux/size.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+# size
+
+> Displays the sizes of sections inside binary files.
+> More information: <https://www.ibm.com/docs/en/aix/7.2?topic=s-size-command>.
+
+- Display the size of sections of object or executable file:
```
Comment (pixelcmtd) [inline pages/linux/size.md:6]:
```suggestion
- Display the size of sections in a given object or executable file:
```

File: pages/linux/size.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+`size {{file}}`
+
+- To display the size of an executable file in [o]ctal:
+
+`size -o {{file}}`
```
Comment (pixelcmtd) [inline pages/linux/size.md:12]:
```suggestion
- To display the size of sections in a given object or executable file in [o]ctal:

`size {{-o|--radix=8}} {{path/to/file}}`
```

File: pages/linux/size.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+- To display the size of an executable file in [d]ecimal:
+
+`size -d {{file}}`
```
Comment (pixelcmtd) [inline pages/linux/size.md:16]:
```suggestion
- To display the size of sections in a given object or executable file in [d]ecimal:

`size {{-d|--radix=10}} {{path/to/file}}`
```

File: pages/linux/size.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+
+- To display the size of an executable file in he[x]adecimal:
+
+`size -x {{file}}`
```
Comment (pixelcmtd) [inline pages/linux/size.md:20]:
```suggestion
- To display the size of sections in a given object or executable file in he[x]adecimal:

`size {{-x|--radix=16}} {{path/to/file}}`
```

**Review by pixelcmtd — APPROVED** [review]

**Comment by suyashbhawsar** [comment]
@pixelcmtd Thanks for the suggestions!

**Review by kbdharun — APPROVED** [review]

File: pages/linux/size.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+# size
+
+> Displays the sizes of sections inside binary files.
+> More information: <https://sourceware.org/binutils/docs/binutils/size.html>.
+
+- Display the size of sections in a given object or executable file:
+
+`size {{path/to/file}}`
+
+- To display the size of sections in a given object or executable file in [o]ctal:
```
Comment (blueskyson) [inline pages/linux/size.md:10]:
```suggestion
- Display the size of sections in a given object or executable file in [o]ctal:
```

File: pages/linux/size.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+`size {{-o|--radix=8}} {{path/to/file}}`
+
+- To display the size of sections in a given object or executable file in [d]ecimal:
```
Comment (blueskyson) [inline pages/linux/size.md:14]:
```suggestion
- Display the size of sections in a given object or executable file in [d]ecimal:
```

File: pages/linux/size.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+`size {{-d|--radix=10}} {{path/to/file}}`
+
+- To display the size of sections in a given object or executable file in he[x]adecimal:
```
Comment (blueskyson) [inline pages/linux/size.md:18]:
```suggestion
- Display the size of sections in a given object or executable file in he[x]adecimal:
```

**Review by blueskyson — APPROVED** [review]

**Comment by suyashbhawsar** [comment]
@blueskyson Thanks for the suggestions!
Can you take a look  at: https://github.com/tldr-pages/tldr/pull/8548 https://github.com/tldr-pages/tldr/pull/8550

**Pull request merged**